content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -27,19 +30,26 @@ When creating SLOs, you can choose from the following types:
+| **[SLO Data Source][5] (up to 15 months of historical data)**         | Available                                                                                                      | Not available                                                                                                                                                       | Available                                             |
+| **Handling missing data in the SLO calculation**                      | Missing data is ignored in SLO status and error budget calculations                                            | Missing data is handled based on the [underlying Monitor's configuration][6]                                                                                        | Missing data is treated as uptime in SLO status and error budget calculations        |
+| **Uptime Calculations**                                               |  N/A                                                                                                           |  Uptime calculations are based on the underlying Monitor <br><br>If groups are present, overall uptime requires *all* groups to have uptime| [Uptime][7] is calculated by looking at discrete time chunks, not rolling time windows<br><br>If groups are present, overall uptime requires *all* groups to have uptime |
+| **[Calendar View][11] on SLO Manage Page**                            | Available                                                                                                      | Not available                                                                                                                                                       | Available                                                                            |
+| **Public [APIs][8] and Terraform Support**                            | Available                                                                                                      | Available                                                                                                                                                           | Available                                                                            |
+
+## Best practices for choosing an SLO Type
+
+- Whenever possible, use Metric-based SLOs. Best practice for SLOs is to have am error budget with a number of events and the SLO is volume weighted.
+- If you want a time based SLI calculation or SLO tracking uptime, use Time slice SLOs since you won’t have to maintain an underlying monitor.
+- If you need non-metric monitors or multiple monitors, then use monitor based SLOs.
+
```

Comment:
@roxanne-moslehi Can you verify the wording in this section?

Reply:
"an error budget"

Reply:
Suggested wording:
- Whenever possible, use metric-based SLOs. It's best practice to have SLOs where the error budget reflects the number of bad events you have left before you breach your SLO. Your SLO calculations will also be volume weighted based on the number of events.
- If, instead, you want an SLO that tracks uptime and uses a time-based SLI calculation, use time slice SLOs. Unlike monitor-based SLOs, time slice SLOs don't require you to maintain an underlying monitor for your SLO.
- Finally, consider monitor-based SLOs for use cases that are not covered by time slice SLOs, which include SLOs based on non-metric monitors or multiple monitors.

Reply:
```suggestion
## Best practices for choosing an SLO Type

- Whenever possible, use Metric-based SLOs. Best practice is for SLOs to be volume weighted and to have an error budget with a number of events.
- If you want a time based SLI calculation or SLO tracking uptime, use Time slice SLOs since you won’t have to maintain an underlying monitor.
- If you need non-metric monitors or multiple monitors, then use monitor based SLOs.

```